coccinelle/glnx-errors.cocci

```diff
@@ -0,0 +1,6 @@
+@@
+expression p;
+@@
+- glnx_set_error_from_errno (p);
+- goto out;
++ return  glnx_throw_errno (p);
```

Comment:
Right, yep.  Though see ⬇️ - I pushed a rewrite of this which covers a lot more.  Here's [the diff it generates](https://paste.fedoraproject.org/paste/l6JxtC6y82HxCKx-NY8AKF5M1UNdIGYhyRLivL9gydE=):  `37 files changed, 461 insertions(+), 869 deletions(-)`